### ISSUE TITLE
What is the "scaled-linear noise scheduler"?

### ISSUE THREAD

**Issue by chenguolin** [SUDO-AI-3D/zero123plus#28]
Great work! You mention in the paper that the original noise schedule in Stable Diffusion is **"scaled-linear schedule"**.

Could you please provide more details about the "scaled-linear schedule"?

I found that Stable Diffusion didn't release their training scripts. Both latent diffusion model (https://github.com/CompVis/latent-diffusion) and Zero123 (https://github.com/cvlab-columbia/zero123) seem to use the **default linear noise schedule**. Did I miss something?

Thank you.
